javascript/QUIZ APP/javascript.js

```diff
@@ -0,0 +1,128 @@
+const quizQuestion = [
+  {
+    id: 1,
+    question: "What is the chemical symbol for water?",
+    options: [
+      { text: "CO2", isCorrect: false },
+      { text: "H2O", isCorrect: true },
+      { text: "O2", isCorrect: false },
+      { text: "CH4", isCorrect: false },
+    ],
+    explanation: "H2O is the chemical symbol for water",
+  },
+  {
+    id: 2,
+    question: "Who wrote the play “Romeo and Juliet”?",
+    options: [
+      { text: "William Shakespeare", isCorrect: true },
+      { text: "Jane Austen", isCorrect: false },
+      { text: "Charles Dickens", isCorrect: false },
+      { text: "Mark Twain", isCorrect: false },
+    ],
+    explanation: "William Shakespeare wrote the play “Romeo and Juliet”",
+  },
+  {
+    id: 3,
+    question: "Which planet is known as the “Red Planet”?",
+    options: [
+      { text: "Mars", isCorrect: true },
+      { text: "Venus", isCorrect: false },
+      { text: "Jupiter", isCorrect: false },
+      { text: "Saturn", isCorrect: false },
+    ],
+    explanation: "Mars is known as the “Red Planet”.",
+  },
+  {
+    id: 4,
+    question: "What is the currency of Denmark?",
+    options: [
+      { text: "Kroner", isCorrect: true },
+      { text: "Pound", isCorrect: false },
+      { text: "Dollar", isCorrect: false },
+      { text: "Rupees", isCorrect: false },
+    ],
+    explanation: "Kroner is the currency of Denmark.",
+  },
+];
+
+// Creating form and styling the form
+const form = document.createElement("form");
+document.body.appendChild(form);
+form.style.alignItems = "center";
+form.style.margin = "2%";
+
+// Creating div elements for questions and answers
+for (let i = 0; i < quizQuestion.length; i++) {
+  if (quizQuestion[i].question.length < 140) {
+    const divQuestion = document.createElement("div");
+    divQuestion.setAttribute("id", "question-" + quizQuestion[i].id);
+    form.appendChild(divQuestion);
+    divQuestion.innerText =
+      quizQuestion[i].id + ". " + quizQuestion[i].question;
+
+    const divAnswers = document.createElement("div");
+    divAnswers.setAttribute("id", "answers");
+    divQuestion.appendChild(divAnswers);
+
+    for (let j = 0; j < quizQuestion[i].options.length; j++) {
+      const button = document.createElement("button");
+      button.setAttribute("class", "btnoptions");
+      button.innerText = quizQuestion[i].options[j].text;
+      button.dataset.isCorrect = quizQuestion[i].options[j].isCorrect;
+
+      divAnswers.appendChild(button);
+      button.style.display = "block";
+      button.addEventListener("click", uponClicking);
+    }
+
+    // Creating button for randomizing the options
+    const buttonRandom = document.createElement("button");
+    buttonRandom.innerText = "Shuffle";
+    buttonRandom.setAttribute("class", "Shuffle");
```

Comment:
Remember naming conventions and use `classList.add('shuffle')` which is made for handling class modifications